lib/resources/example_json/soarian_examples_observation.rb

```diff
@@ -11,7 +11,11 @@ module Resources
       "link": [
         {
           "relation": "self",
-          "url": "base-url/123abc/Observation?patientId=D4292B7B-AEDF-4CEF-B783-BD3AA1B0DD27&_format=json"
+          "url": "base-url/123abc/Observation?patientId=D4292B7B-AEDF-4CEF-B783-BD3AA1B0DD27&_format=json&page=2"
+        },
+		{
```

Comment:
Could you line up these curly braces

Reply:
Formatting has been taken care.

Reply:
Changes are present here. https://github.com/cerner/fhir.cerner.com/pull/232/commits/416d911dabbb8e9fdd2602dbf306ae20ee43e10d